app/src/main/java/org/coepi/android/cen/CenRepo.kt

```diff
@@ -59,7 +63,7 @@ class CenRepo(private val cenApi: CENApi, private val cenDao: RealmCenDao, priva
         if ( ( cenKeyTimestamp == 0 ) || ( roundedTimestamp(curTimestamp) > roundedTimestamp(cenKeyTimestamp) ) ) {
             // generate a new AES Key and store it in local storage
             val secretKey = KeyGenerator.getInstance("AES").generateKey()
-            cenKey = Base64.getEncoder().encodeToString(secretKey.encoded)
+            cenKey = android.util.Base64.encodeToString(secretKey.encoded,android.util.Base64.DEFAULT)
```

Comment:
Replace the import, to not have to write `android.util.Base64` everywhere.

Reply:
I've searched and these are the only occurrences in that brach

Reply:
If you added this to support lower apis, why are you not uploading the lower api?